contracts/protocol/tokenization/StaticATokenLM.sol

```diff
@@ -251,6 +254,11 @@ contract StaticATokenLM is
     return _withdraw(owner, recipient, staticAmount, dynamicAmount, toUnderlying);
   }
 
+  function setBridgeAddresses(address messagingContract_, address l2TokenBridge_) external {
```

Comment:
shouldn't we restrict it to the owner only?

Reply:
Good point, there's no modifier for that, nor owner here. We could alternatively use `onlyProxy`, I'll add it.

Reply:
`onlyProxy` would work if the proxy also implemented this method so that the fallback wouldn't be hit. For now this should be in the constructor and then we need to have a conversation with Andrey about how the proxy's work.